src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.Android.cs

```diff
@@ -427,6 +427,9 @@ void IOnFocusChangeListener.OnFocusChange(View v, bool hasFocus)
 
 		private void ProcessFocusChanged(bool hasFocus)
 		{
+			//We get the view token early to avoid nullvalues when the view has already been detached
+			var viewWindowToken = _textBoxView.WindowToken;
```

Comment:
Is it correct to keep this indefinitely? Is it still valid, eg, after the activity has been destroyed and recreated? (You can test this by pressing the hardware back button to exit the app, then return to it by pressing its icon.)

Reply:
Never mind - I was misreading this, it looks fine.